cypress/support/calendar.js

```diff
@@ -0,0 +1,112 @@
+import localforage from 'localforage';
+import { Link, MultiColumnListCell, Pane, Modal, Button } from '../../interactors';
+
+Cypress.Commands.add('openCalendarSettings', (isLoggedIn = true) => {
+  if (!isLoggedIn) {
+    cy.loginAsAdmin();
+  }
+  cy.visit('settings/calendar');
+});
+
+Cypress.Commands.add('createCalendar', (testCalendarRequestBody, callback = null) => {
+  cy.wrap(localforage.getItem('okapiSess')).then((okapiSess) => {
+    expect(okapiSess).to.have.property('token');
+    cy.request({
+      url: Cypress.env('OKAPI_HOST') + '/calendar/calendars',
+      method: 'POST',
+      body: testCalendarRequestBody,
+      headers: {
+        'x-okapi-tenant': Cypress.env('okapi_tenant'),
+        'x-okapi-token': okapiSess.token
+      },
+      failOnStatusCode: false
+    }).then((response) => {
+      if (response.status === 409) {
+        cy.deleteCalendarUI(testCalendarRequestBody.name);
+        cy.createCalendar(testCalendarRequestBody, callback);
+      } else {
+        expect(response.status).equals(201);
+        if (callback != null) {
+          callback(response);
+        }
+      }
+    });
+  });
+});
+
+Cypress.Commands.add('deleteCalendar', (calendarID, callback = null) => {
+  cy.wrap(localforage.getItem('okapiSess')).then(okapiSess => {
+    expect(okapiSess).to.have.property('token');
+    cy.request({
+      url: Cypress.env('OKAPI_HOST') + '/calendar/calendars/' + calendarID,
+      method: 'DELETE',
+      headers: {
+        'x-okapi-tenant': Cypress.env('OKAPI_TENANT'),
+        'x-okapi-token': okapiSess.token
+      }
+    }).then((response) => {
+      expect(response.status).equals(204);
+      if (callback) { callback(response); }
+    });
+  });
+});
+
+Cypress.Commands.add('deleteCalendarUI', (calendarName) => {
```

Comment:
don't use cypress.comands.add for it please. move it inside fragments and use just common function

e2e/support/fragments/calendar folder